### PR TITLE
Add ImageBuilder commands

### PR DESCRIPTION
**Pull request by qwordy**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### CODE REVIEW

File: src/imagebuilder/azext_imagebuilder/_help.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    type: command
+    short-summary: List all run outputs for the specified Image Template resource
+    examples:
+      - name: Retrieve a list of all outputs created by the last run of an Image Template
+        text: |-
+               az imagebuilder list_run_outputs --resource-group "myResourceGroup" --name \\
+               "myImageTemplate"
+"""
+
+helps['imagebuilder get_run_output'] = """
```
Comment (qwordy) [inline src/imagebuilder/azext_imagebuilder/_help.py]: do not use _, use - instead!

File: src/imagebuilder/azext_imagebuilder/_params.py [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    image_template_name = CLIArgumentType(overrides=name_arg_type,
+                                          help='The name of the image Template', id_part='name')
+
+    with self.argument_context('imagebuilder create') as c:
+        c.argument('resource_group', resource_group_name_type)
+        c.argument('image_template_name', image_template_name)
+        c.argument('location', arg_type=get_location_type(self.cli_ctx),
+                   validator=get_default_location_from_resource_group)
+        c.argument('tags', tags_type)
+        c.argument('customize', id_part=None, help='Specify the properties used to describe the customization steps of the image, like Image source etc')
```
Comment (yungezz) [inline src/imagebuilder/azext_imagebuilder/_params.py]: call out it could be uri of local file/remote uri, or text  directly?

**Comment by yungezz** [comment]
how to send customevent to ai in az cli?

File: src/imagebuilder/azext_imagebuilder/_params.py (hunk 2 above)
Comment (qwordy) [reply src/imagebuilder/azext_imagebuilder/_params.py]: text directly, or @{filename}.
It's a good idea to support URI.

**Comment by qwordy** [comment]
This PR is abandoned. There is already a command group `image template`.

**Pull request closed without merging**